faf.spec.in

```diff
@@ -93,11 +154,7 @@ BuildRequires: python-munch
 BuildRequires: python-dateutil
 BuildRequires: python2-ratelimitingfilter
 BuildRequires: python-simplejson
-BuildRequires: xstatic-patternfly-common
-BuildRequires: nodejs-typeahead.js
-BuildRequires: nodejs-moment
-BuildRequires: js-jquery
-BuildRequires: xstatic-datatables-common
```

Comment:
Why were these packages removed?

Reply:
It is my mistake, I have returned the lines back.